scania-digital-design-system/sdds
# fix(components/header): fix footer overlapping old header sidemenu

**Describe pull-request**  
- In @scania/components: Fixes old footer overlapping side menu.
- In angular sandbox: Changes header so that the bug can be reproduced.

**Solving issue**  
Fixes: #257, https://tegel.atlassian.net/browse/DTS-787 

**How to test**  
1. Checkout the branch locally
1. Make sure you are on node version 16 and you ran npm install in the appropriate folders
1. Go to the sdds root folder
1. Run this and it should hopefully just work™️
```
cd ./components && npm run build && npm link && cd ../sandbox/angular && npm link @scania/components && npm start
```
5. Once the dev server is running in the angular sandbox, go to the path `/short` in the URL bar
6. Open the sidemenu and see it overlapping the footer! 🏁 


**Screenshots**  
Previous issue:
<img width="520" alt="Screenshot 2023-04-26 at 17 03 42" src="https://user-images.githubusercontent.com/8556022/234618530-327171f1-87a0-42d5-81ed-7a569b7a7a8a.png">

After fix:
<img width="645" alt="Screenshot 2023-04-26 at 17 04 37" src="https://user-images.githubusercontent.com/8556022/234618661-fac2ecce-720f-4fe4-863b-c349867c4dd7.png">


**Additional context**  
Could not reproduce the original issue exactly but it is pretty old. This fix should fix it nonetheless.
